src/Protocol/OStatus.php

```diff
@@ -1952,22 +1940,19 @@ private static function entryFooter($doc, $entry, array $item, array $owner, $co
 		$mentioned = [];
 
 		if (($item['parent'] != $item['id']) || ($item['parent-uri'] !== $item['uri']) || (($item['thr-parent'] !== '') && ($item['thr-parent'] !== $item['uri']))) {
-			$parent = q("SELECT `guid`, `author-link`, `owner-link` FROM `item` WHERE `id` = %d", intval($item["parent"]));
+			$parent = item::selectFirst(['guid', 'author-link', 'owner-link'], ['id' => $item["parent"]]);
```

Comment:
Potential typo: Please rename `item` to `Item`.

Reply:
You are right. This will be fixed.